### PR TITLE
Homepage edits

### PR DESCRIPTION
**Pull request by Dindihub**
This PR covers issue #4 

@chalin please review. Thanks

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -33,36 +29,33 @@ developers can adopt into any software update system.
 
 {{% blocks/feature icon="fa-lightbulb" title="Our work" url="docs/" %}}
 
-Discover how TUF works to secure Update systems
+Discover how TUF secures Update systems
```
Comment (chalin) [inline content/en/_index.md]:
```suggestion
Discover how TUF secures update systems
```

File: content/en/_index.md [hunk 2]
```diff
@@ -33,36 +29,33 @@ developers can adopt into any software update system.
...
 
-{{% blocks/lead color="primary" %}}
+{{% blocks/lead color="primary" type="cncf" %}}
 
 **TUF** is a [Cloud Native Computing Foundation](https://www.cncf.io) (CNCF)
-Graduated project and is [used in production](docs/adoptions) by various tech
-companies and open source organizations. A variant of TUF called
-[Uptane](https://uptane.github.io/) is widely used to secure over-the-air
-updates in automobiles
+Graduated project.
```
Comment (chalin) [inline content/en/_index.md]:
```suggestion
[graduated project](https://www.cncf.io/projects).
```

File: content/en/_index.md [hunk 3]
```diff
@@ -33,36 +29,33 @@ developers can adopt into any software update system.
...
+{.cncf}
 
 [cncf]: https://cncf.io
 [cncf logo]: static/img/cncf-white.svg
 [incubating]: https://www.cncf.io/projects/
 
-{{% /blocks/lead %}}
+{{% /blocks/section %}}
+
+.
```
Comment (chalin) [inline content/en/_index.md:61]: I think that this change is a mistake. Shouldn't this be closing a `blocks/lead`?

```suggestion
{{% /blocks/lead %}}
```

**Review by chalin — COMMENTED** [review]
Good fixes.

See inline comments for preliminary feedback.

### Please ensure that you **merge #6 first, and then rebase** the PR to remove the changes introduced by #6 ...

Then (force) push back to this PR so that I can see the clean set of changes.

File: content/en/_index.md (hunk 3 above)
Comment (chalin) [inline content/en/_index.md]:
```suggestion
```

Comment (chalin) [reply content/en/_index.md]: Also, there's something wrong with the blocks:

> image

File: hugo.yaml [hunk 4]
```diff
@@ -89,25 +90,25 @@ params:
...
-        desc: Follow us on Twitter to get the latest news!
-      - name: Stack Overflow
-        url: https://stackoverflow.com/questions/tagged/docsy
-        icon: fa-brands fa-stack-overflow
-        desc: Practical questions and curated answers
+        desc: Sign up for TUF announcements
+      - name: Slack channel
+        url: https://communityinviter.com/apps/cloud-native/cncf
+        icon: fa-brands fa-slack
+        desc: Connect with us on CNCF slack channel tuf!
```
Comment (chalin) [inline hugo.yaml]:
```suggestion
        desc: Connect with us on CNCF Slack channel for TUF
```

File: hugo.yaml [hunk 5]
```diff
@@ -89,25 +90,25 @@ params:
         improve</a>.
   links: # CUSTOMIZE
     user:
-      - name: GitHub Discussions
-        url: https://github.com/google/docsy/discussions
+      - name: GitHub community
+        url: https://github.com/theupdateframework/community
         icon: fa-brands fa-github
         desc: Discussion and help from your fellow users
```
Comment (chalin) [inline hugo.yaml]: This entry doesn't really make sense since there are no discussions enabled over the community repo.

File: hugo.yaml [hunk 6]
```diff
@@ -89,25 +90,25 @@ params:
...
       - name: User mailing list
```
Comment (chalin) [inline hugo.yaml]:
```suggestion
      - name: TUF Google group
```

File: hugo.yaml [hunk 7]
```diff
@@ -89,25 +90,25 @@ params:
...
-        url: https://groups.google.com/forum/#!forum/docsy-users
+        url: https://groups.google.com/forum/?fromgroups#!forum/theupdateframework
```
Comment (chalin) [inline hugo.yaml]:
```suggestion
        url: https://groups.google.com/g/theupdateframework
```

File: hugo.yaml [hunk 8]
```diff
@@ -89,25 +90,25 @@ params:
...
-        desc: Practical questions and curated answers
+        desc: Sign up for TUF announcements
+      - name: Slack channel
+        url: https://communityinviter.com/apps/cloud-native/cncf
+        icon: fa-brands fa-slack
+        desc: Connect with us on CNCF slack channel tuf!
+      - name: Community meetings
+        url: https://www.cncf.io/calendar
+        icon: fa-regular fa-calendar-days
+        desc: Join our community meetings
```
Comment (chalin) [inline hugo.yaml]: I'm not seeing any meetings for TUF in that calendar page. Am I missing something?

File: hugo.yaml [hunk 9]
```diff
@@ -89,25 +90,25 @@ params:
...
     developer:
-      - name: GitHub
-        url: https://github.com/google/docsy
+      - name: TUF GitHub Repo
+        url: https://github.com/theupdateframework
```
Comment (chalin) [inline hugo.yaml]: Link to the website repo

File: hugo.yaml [hunk 10]
```diff
@@ -89,25 +90,25 @@ params:
...
+        icon: fa-brands fa-slack
+        desc: Connect with us on CNCF slack channel tuf!
+      - name: Community meetings
+        url: https://www.cncf.io/calendar
+        icon: fa-regular fa-calendar-days
+        desc: Join our community meetings
     developer:
-      - name: GitHub
-        url: https://github.com/google/docsy
+      - name: TUF GitHub Repo
```
Comment (chalin) [inline hugo.yaml]:
```suggestion
      - name: GitHub repo
```

File: hugo.yaml (hunk 5 above)
Comment (Dindihub) [reply hugo.yaml]: Okay. Which GitHub repo would be best here?

File: hugo.yaml (hunk 8 above)
Comment (Dindihub) [reply hugo.yaml]: I'm also not sure. I found it listed on TUF community page as their calendar https://github.com/theupdateframework/community?tab=readme-ov-file.

 Maybe @lukpueh can elaborate

**Comment by chalin** [comment]
HI. As it says in the comments of https://github.com/Dindihub/docsy-tuf.io/actions/runs/10299082608/job/28505627683?pr=7, you need to run `npm run fix:format` to fix the formatting, and then push the changes.

**Comment by Dindihub** [comment]
The format checks are failing because of this code format. When I run `npm run fix: format` , it moves the CSS class to another line which distorts the styling of the CNCF logo. I don't know how to go about it. What would you advice? 
Screenshot 2024-08-08 121941

**Comment by Dindihub** [comment]
> HI. As it says in the comments of https://github.com/Dindihub/docsy-tuf.io/actions/runs/10299082608/job/28505627683?pr=7, you need to run `npm run fix:format` to fix the formatting, and then push the changes.

@chalin The format checks are failing because of this code format. When I run `npm run fix: format` , it moves the CSS class to another line which distorts the styling of the CNCF logo. I don't know how to go about it. What would you advice? 
Screenshot 2024-08-08 121941

File: hugo.yaml (hunk 8 above)
Comment (chalin) [reply hugo.yaml]: Oh, for now, just link to that page. (We can change it later if/as needed.)

File: content/en/_index.md [hunk 11]
```diff
@@ -31,36 +29,33 @@ developers can adopt into any software update system.
...
+{{% blocks/feature icon="fa-brands fa-github" title="Contribute" url="https://github.com/theupdateframework" %}}
+Start contributing to TUF open source by creating a Pull request on
+[GitHub](https://github.com/theupdateframework)
+
+{{% /blocks/feature %}}
 
 {{% /blocks/section %}}
 
-{{% blocks/lead color="primary" %}}
+{{% blocks/lead color="primary" type="cncf" %}}
```
Comment (chalin) [inline content/en/_index.md]:
```suggestion
{{% blocks/section color="primary" type="cncf" %}}
```

File: content/en/_index.md [hunk 12]
```diff
@@ -31,36 +29,33 @@ developers can adopt into any software update system.
...
+
+**TUF** is a [Cloud Native Computing Foundation](https://www.cncf.io)(CNCF) [graduated project.](https://www.cncf.io/projects)
```
Comment (chalin) [inline content/en/_index.md]:
```suggestion
**TUF** is a [CNCF](https://www.cncf.io) [graduated project](https://www.cncf.io/projects).
```

File: content/en/_index.md [hunk 13]
```diff
@@ -31,36 +29,33 @@ developers can adopt into any software update system.
...
 
-**TUF** is a [Cloud Native Computing Foundation](https://www.cncf.io) (CNCF)
-Graduated project and is [used in production](docs/adoptions) by various tech
-companies and open source organizations. A variant of TUF called
-[Uptane](https://uptane.github.io/) is widely used to secure over-the-air
-updates in automobiles
 
 [![CNCF logo][]][cncf]
+{.cncf}
```
Comment (chalin) [inline content/en/_index.md]: Once you fixed the opening block type to be `blocks/section`, then you don't need this class specifier:

```suggestion
```

**Review by chalin — COMMENTED** [review]
A few more suggestions, and a fix for your Prettier issues :).

File: hugo.yaml (hunk 5 above)
Comment (chalin) [reply hugo.yaml]: Drop this entry. Readers can sign up on Slack and/or the Google group.

**Comment by chalin** [comment]
> @chalin The format checks are failing because of this code format. When I run `npm run fix: format` , it moves the CSS class to another line which distorts the styling of the CNCF logo. I don't know how to go about it. What would you advice? !

See my last review suggestions. They obviate the need for the CSS class, and so will resolve you issue with the formatter.

**Comment by Dindihub** [comment]
Hi @chalin I'm done making the edits. Please review. Thanks

File: content/en/_index.md [hunk 14]
```diff
@@ -31,36 +29,32 @@ developers can adopt into any software update system.
...
-{{% blocks/lead color="primary" %}}
+{{% blocks/section color="primary" type="cncf" %}}
 
-**TUF** is a [Cloud Native Computing Foundation](https://www.cncf.io) (CNCF)
-Graduated project and is [used in production](docs/adoptions) by various tech
-companies and open source organizations. A variant of TUF called
-[Uptane](https://uptane.github.io/) is widely used to secure over-the-air
-updates in automobiles
+**TUF** is a [CNCF](https://www.cncf.io)
+[graduated project](https://www.cncf.io/projects).
```
Comment (chalin) [inline content/en/_index.md]:
```suggestion
[graduated](https://www.cncf.io/projects) project.
```

File: assets/scss/_styles_project.scss [hunk 15]
```diff
@@ -1,13 +1,31 @@
...
+
+    img {
+      width: 20rem;
+      padding-top: 1rem;
+      max-width: 80%;
+    }
+  }
+}
+
+$tuf-blue: #0082ca;
```
Comment (chalin) [inline assets/scss/_styles_project.scss:31]: This variable isn't used. Can it be dropped, or were you planning on using it later?

File: content/en/_index.md [hunk 16]
```diff
@@ -31,36 +29,32 @@ developers can adopt into any software update system.
...
 
-Discover how TUF works to secure Update systems
+Discover how TUF secures update systems
 
 {{% /blocks/feature %}}
 
-{{% blocks/feature icon="fa-brands fa-github" title="Contribute" url="https://github.com/theupdateframework" %}}
-To contribute to The Update Framework, create a Pull request on our
-[GitHub Repositories](https://github.com/theupdateframework)
+{{% blocks/feature icon="fa-solid fa-gear" title="Production Ready" url="/docs/adoptions" %}}
```
Comment (chalin) [inline content/en/_index.md]:
```suggestion
{{% blocks/feature icon="fa-solid fa-gear" title="Production Ready" url="docs/adoptions/" %}}
```

File: hugo.yaml [hunk 17]
```diff
@@ -90,25 +90,21 @@ params:
...
-        url: https://stackoverflow.com/questions/tagged/docsy
-        icon: fa-brands fa-stack-overflow
-        desc: Practical questions and curated answers
+        desc: Sign up for TUF announcements
+      - name: Slack channel
+        url: https://communityinviter.com/apps/cloud-native/cncf
+        icon: fa-brands fa-slack
+        desc: Connect with us on CNCF Slack channel for TUF
+      - name: Community meetings
+        url: https://www.cncf.io/calendar
```
Comment (chalin) [inline hugo.yaml]:
```suggestion
        url: https://github.com/theupdateframework/community?tab=readme-ov-file
```

File: hugo.yaml [hunk 18]
```diff
@@ -90,25 +90,21 @@ params:
...
+        icon: fa-regular fa-calendar-days
+        desc: Join our community meetings
     developer:
-      - name: GitHub
-        url: https://github.com/google/docsy
+      - name: GitHub repo
```
Comment (chalin) [inline hugo.yaml]:
```suggestion
      - name: GitHub repository
```

**Review by chalin — APPROVED** [review]
👍🏻

File: assets/scss/_styles_project.scss (hunk 15 above)
Comment (Dindihub) [reply assets/scss/_styles_project.scss:31]: > This variable isn't used. Can it be dropped, or were you planning on using it later?

I'll use it later. @lukpueh wanted to include the current brand blue color on the [TUF website ](https://theupdateframework.io/) to the New website.

**Pull request merged**